content/blog/2018/egress-https/index.md

```diff
@@ -105,91 +147,194 @@ Now accessing the web page of the application displays the book details without
+    {{< text bash >}}
+    $ kubectl logs $(kubectl get pods -l app=details -l version=v2 -o jsonpath='{.items[0].metadata.name}') istio-proxy | grep googleapis
+    [2018-08-09T11:32:58.171Z] "GET /books/v1/volumes?q=isbn:0486424618 HTTP/1.1" 200 - 0 1050 264 264 "-" "Ruby" "b993bae7-4288-9241-81a5-4cde93b2e3a6" "www.googleapis.com:443" "172.217.20.74:443"
+    EOF
+    {{< /text >}}
+
+    Note the URL path in the log, the path can be monitored and access policies can be applied based on it. To read more
+    about monitoring and access policies for HTTP egress traffic, check [this blog post](https://archive.istio.io/v0.8/blog/2018/egress-monitoring-access-control/#logging).
+
+### Cleanup of TLS origination to a Google Books web service
 
-Another issue is that the egress rules are currently **not a security feature**; they only **enable** traffic to external services. For HTTP-based protocols, the rules are based on domains. Istio does not check that the destination IP of the request matches the _Host_ header. This means that a malicious microservice inside a service mesh could trick Istio to allow traffic to a malicious IP. The attack is to set one of the domains allowed by some existing Egress Rule as the _Host_ header of the malicious request.
-
-Securing egress traffic is currently not supported in Istio and should be performed elsewhere, for example by a firewall or by an additional proxy outside Istio. Right now, we're working to enable the application of Mixer security policies on the egress traffic and to prevent the attack described above.
-
-### No tracing, telemetry and no mixer checks
-
-Note that currently no tracing and telemetry information can be collected for the egress traffic. Mixer policies cannot be applied. We are working to fix this in future Istio releases.
-
-## Future work
+{{< text bash >}}
+$ kubectl delete serviceentry googleapis
+$ kubectl delete destinationrule originate-tls-for-googleapis
+$ kubectl delete -f @samples/bookinfo/networking/virtual-service-details-v2.yaml@
+$ kubectl delete -f @samples/bookinfo/platform/kube/bookinfo-details-v2.yaml@
+{{< /text >}}
 
-In my next blog posts I will demonstrate Istio egress rules for TCP traffic and will show examples of combining routing rules and egress rules.
+### Relation to Istio mutual TLS
 
-In Istio, we are working on making Istio egress traffic more secure, and in particular on enabling tracing, telemetry, and Mixer checks for the egress traffic.
+Note that the TLS origination in this case is unrelated to
+[the mutual TLS](/docs/concepts/security/#mutual-tls-authentication) applied by Istio. The TLS origination for the
+external services will work, whether the Istio mutual TLS is enabled or not. The **mutual** TLS secures
+service-to-service communication **inside** the service mesh and provides each service with a strong identity. In the
+case of the **external services**, you have **one-way** TLS, the same mechanism used to secure communication between a
```

Comment:
The **external services** in this blog were accessed using **one-way** TLS, the same ...